### PR TITLE
update jenkinsfile

### PR DESCRIPTION
**Pull request by ShylajaDevadiga**
What does this PR do:
Updates jenkinfile common for both k3s and rke2 passing product_name as parameter.

### CODE REVIEW

**Comment by ShylajaDevadiga** [comment]
Testing. Not ready for review

File: scripts/Jenkinsfile [hunk 1]
```diff
@@ -41,43 +46,50 @@ node {
...
               userRemoteConfigs: repo
             ])
           }
-          dir ("./") {
+          dir("./") {
             try {
               stage('Configure and Build') {
                 if (env.AWS_SSH_PEM_KEY && env.AWS_SSH_KEY_NAME) {
-                  dir("./tests/acceptance/modules/config/.ssh") {
+                  dir("./modules/config/.ssh") {
```
Comment (fmoral2) [inline scripts/Jenkinsfile:62]: shy , is this path correct?

**Review by fmoral2 — DISMISSED** [review]

File: scripts/Jenkinsfile (hunk 1 above)
Comment (mdrahman-suse) [reply scripts/Jenkinsfile:62]: my $0.02, can we have the `.ssh` folder in the root path? Easier to isolate the keys and ignore `/distros-test-framework/.ssh/`?

Comment (ShylajaDevadiga) [reply scripts/Jenkinsfile]: updated

**Review by fmoral2 — APPROVED** [review]

**Review by mdrahman-suse — APPROVED** [review]
LGTM

**Pull request merged**